packages/fxa-settings/src/index.tsx

```diff
@@ -51,6 +54,13 @@ try {
   const apolloClient = createApolloClient(config.servers.gql.url);
   const appContext = initializeAppContext();
 
+  const View = Storage.isLocalStorageEnabled(window)
```

Comment:
This could be handled as part of `FXA-11486`, but we should add test coverage for this handling.